src/page-multi-facility/FacilityInputForm.tsx

```diff
@@ -120,12 +120,16 @@ const FacilityInputForm: React.FC<Props> = ({ scenarioId }) => {
 
   const save = () => {
     if (facilityName) {
+      // Set observedAt to right now when updating a facility form from this input form
+      const modelUpdate = model[0];
```

Comment:
Do you need to copy the object in order not to accidentally mutate `model[0]` elsewhere?

```javascript
const modelUpdate = Object.assign({}, model[0]);
```

Reply:
I hadn't noticed any adverse effect but I could have missed them. This makes great sense.